### PR TITLE
[gui] add container.viewcount

### PR DESCRIPTION
**Pull request by phil65**
This would allow me to clean skin code up a bit. See https://github.com/xbmc/skin.estuary/commit/84504c0150bc1bc47123a3cf677a7bf74cdf7414
@xhaggi

### CODE REVIEW

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -2004,7 +2006,10 @@ std::string CGUIInfoManager::GetLabel(int info, int contextWindow, std::string *
       {
         const CGUIControl *control = window->GetControl(window->GetViewContainerID());
         if (control && control->IsContainer())
```
Comment (xhaggi) [inline xbmc/GUIInfoManager.cpp]: you missed the enclosing braces

**Comment by phil65** [comment]
@xhaggi 	updated.

**Comment by xhaggi** [comment]
looks good now.

**Comment by phil65** [comment]
jenkins build this please.

**Pull request merged**